modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer.js

```diff
@@ -128,7 +127,6 @@ const defaultProps = {
   wireframe: false,
   // Optional material for 'lighting' shader module
   material: defaultMaterial,
-  sizeScale: {type: 'number', value: 1, min: 0},
```

Comment:
Was this removed at some point from the shader? I thought we wanted to keep it as a way to change size via a uniform.

Reply:
we have another sizeScale in the same 'defaultProps', so they are duplicated.